src/components/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import HeaderWithCloseButton from './HeaderWithCloseButton';
+import Text from './Text';
+import Modal from './Modal';
+import CONST from '../CONST';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+
+const propTypes = {
+    /** prop to fetch screen width */
+    ...windowDimensionsPropTypes,
+
+    /** props to fetch translation functions */
+    ...withLocalizePropTypes,
+};
+
+class KeyboardShortcutsModal extends React.PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isOpen: false,
+        };
+    }
+
+    componentDidMount() {
+        const shortcutModifiers = KeyboardShortcut.getShortcutModifiers(['control']);
+        this.unsubscribeShortCutModal = KeyboardShortcut.subscribe('?', () => {
+            this.toggleKeyboardShortcutModal(true);
+        }, 'openShortcutDialog', shortcutModifiers, true);
+    }
+
+    componentWillUnmount() {
+        if (!this.unsubscribeShortCutModal) {
+            return;
+        }
+        this.unsubscribeShortCutModal();
+    }
+
+    /**
+     * Set flag for model visibility
+     * @param {Boolean} flag
+     */
+    toggleKeyboardShortcutModal(flag) {
```

Comment:
Minor detail, but I don't think this function is really useful. I would create two functions – one called `showKeyboardShortcutsModal` and another called `hideKeyboardShortcutsModal`, bind them both to `this` in the constructor, then use each accordingly.